index.js

```diff
@@ -236,6 +236,12 @@ class NYC {
     return this._transform(code, filename)
   }
 
+  maybePurgeSourceMapCache () {
+    if (!this.cache) {
```

Comment:
Maybe I'm crazy but isn't this clause written backwards? My guess is that it should be `!!this.cache` but I don't know for sure, just wanted to ask to be sure.

Reply:
The code is correct.  lib/source-map.js needs to cache source-maps in memory during execution of nyc.  But if caching is disabled then we need to purge the caches before running reports.

Reply:
Ah. The naming got me.